.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,41 @@
+name: Coding Standards
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  phpcs:
+    name: PHP coding standards
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
```

Comment:
I think web need some PHP versions that this phpColors requires.

That is, we should test the `php-7.2`, `php-7.3`, `php-7.4`, `php-8.0` and `php-8.1` versions.

Reply:
Sorry. My fault. This is for coding standards. Above issue is available on the `php-tester.yml`.